### PR TITLE
Update uwsgi stack for Stretch

### PR DESCRIPTION
**Pull request by ocdtrekkie**
This PR isn't complete yet, but is currently "better than what currently happens". It updates the MySQL install behavior from my PHP PR (#222), so that we don't install MariaDB.

Current issue using python test app:

> /opt/app/env exists, moving on
/opt/app/.sandstorm/build.sh: line 11: /opt/app/env/bin/pip: No such file or directory
Connection to 127.0.0.1 closed.
[31mCommand failed with a non-zero exit status (127).[0m

I suspect pip needs to be installed somehow, as it is indeed not in /opt/app/env/bin.

### CODE REVIEW

**Comment by kentonv** [comment]
To be clear, do you want me to merge this as-is or do you want me to wait until it is more complete?

**Comment by ocdtrekkie** [comment]
At some point I intend to finish this, but it definitely does not work without this change as vagrant-spk's adjustments to MySQL do not work on MariaDB. This is arguably "less broken" than what is currently in master, and gets someone closer to a working build should they try to use it.

tl;dr: Master is broken and this is less broken, no reason not to merge it for now, saves me having to re-add it manually when testing.
